notifications_utils/template.py

```diff
@@ -364,6 +361,31 @@ def __str__(self):
 class BaseBroadcastTemplate(BaseSMSTemplate):
     template_type = 'broadcast'
 
+    MAX_CONTENT_COUNT_GSM = 1_395
+    MAX_CONTENT_COUNT_USC2 = 615
```

Comment:
Minor thing - it's [UCS2](https://en.wikipedia.org/wiki/Universal_Coded_Character_Set), not USC2

Reply:
Nice one, I squashed it out